### PR TITLE
Added newly generated STM Files

### PR DESCRIPTION
**Pull request by RajeshJITX**
These files are generated using new code. First pass to make sure this checks pass

### CODE REVIEW

**Comment by bpellenard** [comment]
Are there some files that are not auto-generated in this PR?

**Comment by RajeshJITX** [comment]
> Are there some files that are not auto-generated in this PR?

No. All the files are generated. Mixing them would make it very difficult to review

File: components/st-microelectronics/STM32F030C6T6.stanza [hunk 1]
```diff
@@ -0,0 +1,91 @@
+
```
Comment (bpellenard) [inline components/st-microelectronics/STM32F030C6T6.stanza:1]: Add auto-generation info?

File: components/st-microelectronics/STM32F030C8T6.stanza [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  import ocdb/property-structs
+
+public pcb-component component :
+  manufacturer = "STMicroelectronics"
+  mpn = "STM32F030C8T6"
+  val generic-props = GenericPin(Interval(-0.3, 4.0, false), 2000.)
+  val power-props = PowerPin(Interval(2.4, 3.6, false))
+  pin-properties :
+    [pin:Ref                   | pads:Int ... | side:Dir| generic-pin:GenericPin | power-pin:PowerPin ]
+      [VDD[0]   | 1 | Up | - | - ]
```
Comment (bpellenard) [inline components/st-microelectronics/STM32F030C8T6.stanza:26]: indentation is different from the rest

File: components/st-microelectronics/STM32F030C8T6.stanza [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    [PB[7]   | 43 | Right | - | - ]
+    [BOOT[0]   | 44 | Left | - | - ]
+    [PB[8]   | 45 | Right | - | - ]
+    [PB[9]   | 46 | Right | - | - ]
+
+  ;eval-when has-property?(VDD.rail-voltage):
+  ;  val vdd = property(VDD.rail-voltage)
+  ;  val generic-props-FT = GenericPin(Interval(-0.3, 4.0 + vdd, false), 2000)
+  ;  for p in FT-pins do :
+  ;    property(p.generic-pin) = generic-props-FT
```
Comment (bpellenard) [inline components/st-microelectronics/STM32F030C8T6.stanza:79]: do we want to keep this commented code?

File: components/st-microelectronics/STM32F030K6Tx.supports.stanza [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+      gpio.gpio => self.PA[4]
+
+    supports gpio:
+      gpio.gpio => self.PA[5]
+
+    supports gpio:
+      gpio.gpio => self.PA[6]
+
+    supports gpio:
+      gpio.gpio => self.PA[7]
```
Comment (bpellenard) [inline components/st-microelectronics/STM32F030K6Tx.supports.stanza:46]: I would really like to be able to do:
```
supports gpio:
        gpio.gpio => self.PA[1 through 7]
```

**Comment by RajeshJITX** [comment]
Too big to merge. Will get same functionality using MongoDB

**Pull request closed without merging**